org.eclipse.buildship.core.test/src/main/groovy/org/eclipse/buildship/core/util/gradle/GradleDistributionTest.groovy

```diff
@@ -73,7 +72,7 @@ class GradleDistributionTest extends Specification {
         1 * connector.useBuildDistribution()
     }
 
-    def "toString"() {
+    def "_toString"() {
```

Comment:
Why not give these methods a more descriptive name?

Reply:
This class was just moved from tooling-commons. The change you see here is because the groovy tooling showed some errors.

Reply:
I know and my comment was that instead of making the test names even more artificial, we should make them more human friendly.

Reply:
Done